corehq/apps/callcenter/views.py

```diff
@@ -40,6 +41,7 @@ def static_options(self):
         return []
 
 
+@location_safe
```

Comment:
> My understanding is that that method is whats needed to make a view "location safe"?

For it to be location safe, users, locations, cases, and forms should be restricted for users whose access is restricted.  I think the parent class of this restricts those, so this may be fine.

I was more asking about why it's location safe.  Location-restricted users have access to only a limited, whitelisted portion of HQ (I'll try to set up a demo of that soon), where everything is restricted based on their assigned location.  We're slowly expanding what's available in that whitelist, so I was wondering if this is part of a page or workflow that'll be available.